recipes/dataframe/all/conanfile.py

```diff
@@ -65,7 +70,8 @@ def configure(self):
             "Visual Studio": "15",
             "gcc": "7",
             "clang": "6",
-            "apple-clang": "9.0",
+            "apple-clang": "10.0",
```

Comment:
Here is my above
```suggestion
            "apple-clang": "10.0" if tools.Version(self.version) >= "1.12.0" else "9.0",
```

Reply:
@prince-chrismc , the problem is that `apple-clang 9.0` has some issues with finding header file `any`. That is why I changed it in the first place. Version 9.0 would not compile my code

Reply:
The compiler requirement might have increased for the new version, but there is no reason to artificial increase the compiler requirement for the old version. Please consider @prince-chrismc's suggestion

Reply:
done